metrics/web/test/auth/presentation/widgets/auth_form_test.dart

```diff
@@ -53,12 +48,8 @@ void main() {
     }
 
     setUp(() {
-      authNotifier = AuthNotifier(
-        receiveAuthUpdates,
-        signInUseCase,
-        googleSignInUseCase,
-        signOutUseCase,
-      );
+      authNotifier = AuthNotifierMock();
```

Comment:
Should we add a `tearDown` to reset the auth notifier mock after each test to make sure that the tests do not influence each other? Also, if we'll do it, we'll be able to avoid creation of the separate notifier in all tests.